src/hooks/Utils.tsx

```diff
@@ -42,6 +43,22 @@ export const useDebounce = <Type extends unknown>(value: Type, delay: number) =>
   return debouncedValue;
 };
 
+export const removeIdsFromFields = (fields: Array<TextFormField | SelectFormField>) => {
```

Comment:
Dette er ikke en hook så jeg tror den passer bedre i utils.tsx filen i root. Også hadde det vært lurt med JsDoc i tillegg for den:
```
/**
  * Removes id's from fields and the options of the given fields
  * 
  * @param fields The fields to remove the id's from
*/
```